clients/unshaded/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -138,6 +141,34 @@
   }
 
   /**
+   * Only called by {@link getDummyStream}.
+   * An alternative to construct a RemoteBlockInstream, bypassing all the checks.
+   * The returned RemoteBlockInStream can be used to call {#link #readRemoteByteBuffer}.
+   *
+   * @param file, any, could be null
+   * @param readType, any type
+   * @param blockIndex, any index
+   * @param ufsConf, any ufs configuration
+   * @param tachyonConf, any
+   * @param addFlag, add another field so that this constructor differentiates
+   */
+  private RemoteBlockInStream(TachyonFile file, ReadType readType, int blockIndex, Object ufsConf,
+      TachyonConf tachyonConf, boolean addFlag) {
+    super(file, readType, blockIndex, tachyonConf);
+  }
+
+  /**
+   * Return a dummy RemoteBlockInStream object by calling the private constructor.
```

Comment:
Not clear why such a `dummyStream` is even useful and how is it used in the current javadoc.